embulk-input-s3/src/test/java/org/embulk/input/s3/TestS3InputStreamReopener.java

```diff
@@ -89,6 +90,28 @@ public void reopenS3FileByReopener_on_retry_gave_up_should_throw_original_except
         opener.reopen(0, new RuntimeException());
     }
 
+    @Test(expected = AmazonClientException.class)
+    public void reopenS3FileByReopener_on_retry_always_throw_exception()
+            throws Exception
+    {
+        { // always failed call with 2 retries
```

Comment:
This extra block is unnecessary, `reopenS3FileByReopener` has to blocks to avoid accidental scope leak/shadow (or improve readability), but I believe it is totally unnecessary for a method to have a single nested code block.

Reply:
Typo mistake, fixed.